spec/features/create_object_h2_spec.rb

```diff
@@ -87,8 +87,7 @@
 
     # Opens Argo and searches on title
     visit Settings.argo_url
-    fill_in 'Search...', with: item_title
-    click_button 'Search'
+    find_field('Search...').send_keys(item_title, :enter)
```

Comment:
[`fill_in`](https://www.rubydoc.info/github/jnicklas/capybara/Capybara/Node/Actions:fill_in) returns the element that was filled in, so if people think it reads better, i could instead do something like:

```ruby
search_entry_field = fill_in 'Search...', with: item_title
search_entry_field.send_keys(:enter)
```

the first param to `send_keys` here just inputs the given string as text.  the second param causes an `Enter` key press.  ([`send_keys`](https://www.rubydoc.info/github/jnicklas/capybara/Capybara%2FNode%2FElement:send_keys) is pretty nice, it takes an arbitrary list of params, representing a sequence of keystrokes, and a simultaneous key combo can be given by an array param, e.g. `[:control, 'a']` to select all text in a textbox (depending on the OS, for that last example).